documentation/getting-started.md

```diff
@@ -18,35 +18,21 @@ We provide this in a number of ways:
 * Cryptographic identity (x509 certificate issued from our CA)
 * Enabling the removal of hard coded credentials from web applications and firmware
 
-You can access all these features by installing the [WoTT Agent](https://github.com/WoTTsecurity/agent) and using the [WoTT Dashboard](https://dash.wott.io) to manage your devices through a simple and user-friendly interface. We'll walk you through step-by-step in setting up and installing the WoTT agent so you can start to implement hassle-free security on your IoT or edge devices.
```

Comment:
Remove link to github. send people to the registration page in both cases. We want to drive sign ups as the call to action.